MANIFEST.in

```diff
@@ -1,2 +1,19 @@
+exclude docs
```

Comment:
does one have to explicitly exclude directories?

Reply:
Yes I've always found this to be the case

Reply:
For example, we don't do this for AiiDA https://github.com/aiidateam/aiida-core/blob/develop/MANIFEST.in
and the docs are not included in the pypi package https://pypi.org/project/aiida-core/1.6.3/#files

Reply:
do actually you are right, I removed it and check-manifest did not complain (could have sworn I had an issue with this before)

Reply:
no removed 👍